lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -176,19 +173,16 @@ public void testClaimTaskWithDefaultFlag()
         assertNotNull(createdTask);
         assertNull(createdTask.getClaimed());
 
-        Instant before = Instant.now();
-        waitAMillisecond();
+        Instant before = createdTask.getCreated();
         Task claimedTask = taskService.claim(createdTask.getId());
-        waitAMillisecond();
-        Instant after = Instant.now();
 
         assertNotNull(claimedTask.getOwner());
         assertEquals(claimedTask.getOwner(), CurrentUserContext.getUserid());
         assertNotNull(claimedTask.getClaimed());
-        assertTrue(claimedTask.getClaimed().isAfter(before));
-        assertTrue(claimedTask.getClaimed().isBefore(after));
-        assertTrue(claimedTask.getModified().isAfter(before));
-        assertTrue(claimedTask.getModified().isBefore(after));
+        assertTrue(isBeforeOrEqual(before, claimedTask.getClaimed()));
+        assertTrue(isBeforeOrEqual(claimedTask.getCreated(), claimedTask.getClaimed()));
+        assertTrue(isBeforeOrEqual(claimedTask.getClaimed(), Instant.now()));
```

Comment:
You are using before / claimedTask.getCreated() not consistant. Please use the before variable

Reply:
nevermind.